actiontext/test/test_helper.rb

```diff
@@ -45,4 +45,11 @@ def create_file_blob(filename:, content_type:, metadata: nil)
     end
 end
 
+# Encryption
+ActiveRecord::Encryption.configure \
+  master_key: "test master key",
```

Comment:
Considering [this issue](https://github.com/rails/rails/issues/33677) it would be great to rename it to something like `encryption_key` instead.

Reply:
Renaming it to `primary_key` [here](https://github.com/basecamp/rails/commit/ada6c39708ffd0d47735de9eaeb82734d875efc6). Thanks for the heads up @achempion!